### ISSUE TITLE
Demo | Assistive technology does not read the access link for each product in demo (End: 6/12)

### ISSUE THREAD

**Issue by msj0nes** [department-of-veterans-affairs/caseflow#14377]
# Accessibility Issue Description

#### Defects Found: 
_Defect 1_: Information, structure and relationships conveyed through presentation cannot be understood. 

#### Details 
If a user selects one of the application elements such as Queue, a link is displayed visually under the list of applications that reads: Your Queue. Assistive technology users do not know this link is displayed after the list of applications and must be activated in order to view the Queue Caseflow application. (Not Resolved: 12/13/2019) (Not Resolved 3/26/2020)

 
![image](https://user-images.githubusercontent.com/41164406/82701876-f32dec80-9c3e-11ea-92f6-346bc93057d5.png)

## Acceptance criteria
- [ ] Testing should conclude the each access link is read by assistive technology.

**Comment by jcq** [comment]
Best I can tell, this is an issue with accessibility on the `TabWindow` component. It will require refactoring to match the best practices for accessibility for tabs. 

This will involve better ARIA labeling/linking between tabs and the tab (content) panes, and implementing keyboard navigation (along with adjusting tabindex prop so that hitting tab goes to the tab content/pane).

### Background
- [MDN on tabs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role)
- [React Bootstrap implementation](https://react-bootstrap.github.io/components/tabs/)
- [React Bootstrap code](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Tabs.js)
- [Material UI implementation](https://material-ui.com/components/tabs/#keyboard-navigation)
